<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Import Data</class>
 <widget class="QMainWindow" name="Import Data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="arrBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>761</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Range</string>
    </property>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>최소 범위</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="minText">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>최대 범위</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>40</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>범주 간격</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="maxText">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="interText">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="runButton">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>40</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>761</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>530</y>
      <width>691</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>&lt; back</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string>next &gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="finishButton">
       <property name="text">
        <string>finish</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>761</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>551</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Non-Identifier: Categorical</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="arrBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>371</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Original</string>
    </property>
    <widget class="QTableWidget" name="original">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>31</y>
       <width>341</width>
       <height>241</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>15</number>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="arrBox_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>230</y>
      <width>371</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Categorical</string>
    </property>
    <widget class="QTableWidget" name="categorical">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>341</width>
       <height>241</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>15</number>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
